test/cpp/interop/stress_test.cc

```diff
@@ -199,23 +214,48 @@ int main(int argc, char** argv) {
   LogParameterInfo(server_addresses, tests);
 
   WeightedRandomTestSelector test_selector(tests);
+  MetricsServiceImpl metrics_service;
 
   gpr_log(GPR_INFO, "Starting test(s)..");
 
   vector<thread> test_threads;
```

Comment:
I know the "using std::thread" predates this PR, but maybe now would be a good time to fix it? Can we switch to explicitly saying "std::thread", or better "grpc::thread" throughout? (Actually, I'd probably prefer std::vector etc too since most of the other tests are written that way.)